content/blog/2018/01/2018-01-05-gsoc2018-call-for-mentors.adoc

```diff
@@ -0,0 +1,92 @@
+---
+:layout: post
+:title: "Google Summer Of Code 2018: Call for mentors"
+:tags:
+- gsoc
+- gsoc2018
+- events
+- community
+:author: oleg-nenashev
+---
+
+This year we are interested to participate in
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code (GSoC)].
+As in 2016/2017, we are looking for mentors.
+So yes, we are looking for you :)
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+== What is GSoC?
+
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code]
+(GSoC) is an annual, international, program which encourages
```

Comment:
"annual international program"

Reply:
I copy-pasted it from somewhere, will fix that